ci/kokoro/docker/build-in-docker-quickstart-bazel.sh

```diff
@@ -59,43 +59,52 @@ if [[ -r "/c/kokoro-run-key.json" ]]; then
   )
 fi
 
-build_service() {
-  local -r service="$1"
+build_quickstart() {
+  local -r library="$1"
 
-  echo "================================================================"
-  log_yellow "building ${service}"
-  ( cd "google/cloud/${service}/quickstart";
-    log_normal "capture bazel version"
-    ${BAZEL_BIN} version
-    log_normal "fetch dependencies for ${service}'s quickstart"
-    "${PROJECT_ROOT}/ci/retry-command.sh" \
-        "${BAZEL_BIN}" fetch -- ...
+  cd "${PROJECT_ROOT}/google/cloud/${library}/quickstart"
```

Comment:
Perhaps a function should never have a `cd` side effect?  Given that we're in `bash`, `pushd` and `popd` may be appropriate.